src/css/custom.css

```
@@ -1,30 +1,225 @@
+}
+
+.alert--info {
+  --ifm-alert-background-color: rgba(242, 140, 56, 0.1);
+  --ifm-alert-color: var(--ifm-color-primary);
+  border-left-color: var(--ifm-color-primary);
+}
+
+.alert--warning {
+  --ifm-alert-background-color: rgba(255, 107, 107, 0.1);
+  --ifm-alert-color: #ff6b6b;
+  border-left-color: #ff6b6b;
```

Stylelint (17.2.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)